verification/certifyKey.go

```diff
@@ -120,18 +123,23 @@ const (
 
 type TcgMultiTcbInfo = []DiceTcbInfo
 
+type CertifyKeyParams struct {
+	Label []byte
+	Flags CertifyKeyFlags
+}
+
```

Comment:
This struct declaration is moved out of testCertifyKey function to reduce the function size.